pkg/crd/generator/testData/.gitignore

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Why we need this file?
Is this file scaffolded?

Reply:
Yes. `testData` is a Kubebuilder generated project.

Reply:
removed it.